.github/workflows/js-version.yml

```
@@ -34,4 +39,8 @@ jobs:
           git config user.email github-actions@github.com
           yarn version --${{ inputs.VERSION_TYPE }}
```

yarn have config command to set the git prefix
```suggestion
          if [ ${{ inputs.VERSION_PREFIX }} = true ]; then
              TAG_PREFIX="v"
          else
              TAG_PREFIX=""
          fi
          yarn config set version-tag-prefix "$TAG_PREFIX"
          yarn version --${{ inputs.VERSION_TYPE }}
```
then you do not need parsing substring